apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -0,0 +1,316 @@
+- **Certificate**  
+  Supabase Auth (the Service Provider) trusts assertions from an Identity Provider based on the signature attached to the assertion. The signature is verified according to the certificate present in the Metadata.
+- **ACS URL**  
+  This is one of the most important SAML URLs. It is the URL where Supabase Auth will accept assertions from an identity provider. Basically, once the identity provider verifies the user's identity it will redirect to this URL and the redirect request will contain the assertion.
+- **Binding (Redirect, POST or Artifact)**  
+  This is a description of the way an identity provider communicates with Supabase Auth. When using the Redirect binding, the communication occurs using HTTP 301 redirects. When it's `POST`, it's using `POST` requests sent with `<form>` elements on a page. When using Artifact, it's using a more secure exchange over a Redirect or `POST`.
+- **RelayState**  
+  State used by Supabase Auth to hold information about a request to verify the identity of a user.
+
+## Important SAML 2.0 information
+
+Below is information about your project's SAML 2.0 configuration which you can share with the company or organization that you're trying to on-board.
+
+| Name         | Value                                                     |
+| ------------ | --------------------------------------------------------- |
+| EntityID     | `https://<project>.supabase.co/auth/v1/sso/saml/metadata` |
+| Metadata URL | `https://<project>.supabase.co/auth/v1/sso/saml/metadata` |
+| ACS URL      | `https://<project>.supabase.co/auth/v1/sso/saml/acs`      |
+| SLO URL      | `https://<project>.supabase.co/auth/v1/sso/slo`           |
+| NameID       | Required `emailAddress` or `persistent`                   |
+
+Note that SLO (Single Logout) is not supported at this time with Supabase Auth as it is a rarely supported feature by identity providers. However the URL is registered and advertised for when this does become available.
+
+Append `?download=true` to the Metadata URL to get a downloadable Metadata XML file.
+
+### User accounts and identities
+
+User accounts and identities created via SSO differ from regular (email, phone, password, social login...) accounts in these ways:
+
+- **No automatic linking.**  
+  Each user account verified using a SSO identity provider will not be automatically linked to existing user accounts in the system. That is, if a user `jane.doe@company.com` had signed up with a password, and then uses their company SSO login with your project, there will be two `jane.doe@company.com` user accounts in the system.
+- **Emails are not necessarily unique.**  
+  Given the behavior with no automatic linking, email addresses are no longer a unique identifier for a user account. Please always use the user's UUID to correctly reference user accounts.
```

Comment:
Good shout. In the future, we can also consider highlighting this in other Auth sections too, perhaps in the [Email login page] or similar (https://supabase.com/docs/guides/auth/auth-email). Imagine that quite a few Auth users might assume email uniqueness but won't be using SSO.